settings/rest-api.php

```diff
@@ -323,6 +357,66 @@ function register_user_fields(): void {
 		]
 	);
 
+	register_rest_field(
+		'user',
+		'svn_password_required',
+		[
+			'get_callback' => function( $user ) {
+				global $wpdb;
+
+				$user = get_userdata( $user['id'] );
+				if ( ! $user ) {
+					return false;
+				}
+
+				// Temporarily only show this to 2FA beta-testers, until the systems side is handled.
+				if ( ! is_2fa_beta_tester( $user ) ) {
+					return false;
+				}
+
+				// Committers, supes, etc. It's likely these users will need a SVN password.
+				if ( function_exists( 'is_special_user' ) && is_special_user( $user->ID ) ) {
+					return true;
+				}
+
+				// Plugin committers & Theme authors have this user meta set.
+				if ( $user->has_plugins || $user->has_themes ) {
+					return true;
+				}
+
+				return false;
+			},
+			'schema' => [
+				'type'    => 'boolean',
+				'context' => [ 'edit' ],
+			]
+		]
+	);
+
+	register_rest_field(
+		'user',
+		'svn_password_created',
+		[
+			'get_callback' => function( $user ) {
+				// Local environment doesn't have the SVN password system, just return false for that.
+				if ( ! function_exists( 'WordPressdotorg\Security\SVNPasswords\get_svn_password_creation_date' ) ) {
+					return false;
```

Comment:
Since this returns `false`, it doesn't allow us to test locally because of [this check](https://github.com/WordPress/wporg-two-factor/pull/280/files#diff-e0a1347d34330bff3a6fb9582b587a9cef40122e84ad844ae7e290311d5c0dbcR77). Can we return a hard-coded date for testing?

Reply:
Yep, but not really at the same time.

You can return Dummy data here, but if you hard-code dummy data you can't then test the branch where you've not set a password.

The best option would probably be to add a local test variant of the svn password functionality that uses user meta.